pkgs/servers/http/couchdb/3.0.0.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv, fetchurl, erlang, icu, openssl, spidermonkey
+, coreutils, bash, makeWrapper, python3 }:
+
+stdenv.mkDerivation rec {
+  pname = "couchdb";
+  version = "3.1.0";
+
+
+  # when updating this, please consider bumping the OTP version
```

Comment:
what do you mean by OTP?

Reply:
This is from the 2.0.0.nix and is referring to the erlang/otp version. I removed it entirely since the current version for erlang in nixos is 22 and couchdb3 accepts 22.X, but I will hardcode it to `erlang = erlangR22` since couchdb3's pinning is unlikely to keep up with nix's default erlang.